xml/System.Threading.Tasks/ValueTask`1.xml

```diff
@@ -661,9 +661,13 @@ As such, the default choice for any asynchronous method should be to return a <x
         <remarks>
           <format type="text/markdown"><![CDATA[  
 ## Remarks  
+ This method is intended for compiler use rather than for use in application code.
```

Comment:
Why was this added?

Reply:
I inferred this from the "short rule" in [your blog post here](https://devblogs.microsoft.com/dotnet/understanding-the-whys-whats-and-whens-of-valuetask/)

> with a ValueTask or a ValueTask<TResult>, you should either await it directly (optionally with .ConfigureAwait(false)) or call AsTask() on it directly, and then never use it again

So `Result` is off the beaten path. Especially with the newer `IValueTaskSource` scenario, where `Result` should only be called after you (somehow) know the `ValueTask` is completed. So the use case for `ValueTask<T>.Result` is much smaller than that of `Task<T>.GetAwaiter().GetResult()` (which is marked with this same warning).

That said, I have no problem removing this.

Reply:
I think we should remove it.  It's fine to warn people off from using it (i.e. "just use await"), but the C# compiler doesn't actually use this property :)